hieradata/site/tu/cluster/auxtel-ccs.yaml

```diff
@@ -12,4 +12,6 @@ clustershell::groupmembers:
   all: {group: "all", member: "auxtel-fp01,auxtel-mcm"}
 
 postfix::relayhost: "[mail.lsst.org]:submission"
-ccs_monit::mailhost: "mail.lsst.org"
+ccs_monit::mailhost:
+  - "mail.lsst.org"
```

Comment:
Why is tu still listing an explicit mail relay?

Reply:
Because it is working with the current configuration, so I left it alone (due to changes in the module, the hiera has to change to preserve the status quo).

Reply:
Shouldn't it work using the localhost for relay too?  It seems like this is a configuration difference between a test stand and the summit that could be eliminated.

Reply:
I'll try it out using localhost and report back.

Reply:
I have verified that just using localhost in site/tu works fine too.
I have forced pushed a corresponding new version, please review when convenient, thanks.